### PR TITLE
ci: migrate to codecov

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [x] Build or CI related changes
- [ ] Documentation content changes

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by waterplea — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 [![All packages CI](https://github.com/tinkoff/taiga-ui/actions/workflows/build.yml/badge.svg?branch=main)](https://github.com/tinkoff/taiga-ui/actions/workflows/build.yml)
 [![Discord](https://img.shields.io/discord/748677963142135818?color=7289DA&label=%23taiga-ui&logo=discord&logoColor=white)](https://discord.gg/Us8d8JVaTg)
 [![angular-open-source-starter](https://img.shields.io/badge/made%20with-angular--open--source--starter-d81676?logo=angular)](https://github.com/tinkoff/angular-open-source-starter)
+[![codecov](https://codecov.io/gh/Tinkoff/taiga-ui/branch/main/graph/badge.svg?token=4A9ZQ0T7S5)](https://codecov.io/gh/Tinkoff/taiga-ui)
```
Comment (MarsiBarsi) [inline README.md:7]: what is the final number there? are we sure that we want to share it in readme now?

Comment (waterplea) [reply README.md:7]: I believe it's 60 something. Yeah, let's not show it for now and once we have a correct measure we can work on improving the number

Comment (splincode) [reply README.md:7]: Okey) final number - 66.99%

image

Comment (splincode) [reply README.md]: fixed

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**